File: sub-mysql-web/src/main/java/org/github/base/service/impl/ArticleTagServiceImpl.java
package org.github.base.service.impl;

import org.github.base.entity.ArticleTagEntity;
import org.github.base.mapper.IArticleTagMapper;
import org.github.base.service.IArticleTagService;
import org.github.base.ServiceImpl;
import org.springframework.stereotype.Service;

/**
 * <p>
 *  服务实现类
 * </p>
 *
 * @author JYD_XL
 * @since 2021-05-20
 */
@Service
public class ArticleTagServiceImpl extends ServiceImpl<IArticleTagMapper, ArticleTagEntity> implements IArticleTagService {}

File: sub-mysql-blog-service/src/main/java/org/github/mysql/blog/base/service/impl/ArticleTagServiceImpl.kt
@file:Suppress("RedundantSemicolon")

package org.github.mysql.blog.base.service.impl;

import org.github.mysql.blog.base.entity.ArticleTagEntity;
import org.github.mysql.blog.base.mapper.IArticleTagMapper;
import org.github.mysql.blog.base.service.IArticleTagService;
import org.github.base.ServiceImpl;
import org.springframework.stereotype.Service;

/**
 *  服务实现类
 *
 * @author JYD_XL
 */
@Service
open class ArticleTagServiceImpl : ServiceImpl<IArticleTagMapper, ArticleTagEntity>(), IArticleTagService
